addons/medical/stringtable.xml

```diff
@@ -3474,7 +3485,7 @@
             <Russian>Настройка лечения в медицинской системе ACE</Russian>
             <Polish>Skonfiguruj zaawansowane ustawienia leczenia systemu medycznego ACE</Polish>
             <Spanish>Configure las opciones de tratamiento del ACE Médico</Spanish>
-            <German>Behandlungseinstellungen vom ACE-Medical konfigurieren</German>
+            <German>Behandlungseinstellungen vom ACE-Medizin konfigurieren</German>
```

Comment:
Wenn ich dies ändern wollte, so müsste ich auch diverese andere Namen in anderen Dateien ändern. Ansonsten bestünde keine Homogenität innerhalb der Bergriffsverwendungen.